core/utils/collection.php

```diff
@@ -118,6 +118,12 @@ public function map( callable $callback ) {
 		return new static( array_combine( $keys, $items ) );
 	}
 
+	public function each( callable $callback ) {
```

Comment:
Maybe add a PHPDoc, just for consistency with other functions

Reply:
Done.